documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md

```diff
@@ -221,18 +221,21 @@ The final version of the evaluation report may only be submitted to the Applican
 8. The Applicant may only make the evaluation report available to third parties in full text, stating the date of issue, and pass it on after written approval by the Evaluation Facility.
 
 ### 3.6 Change Certification     
-1. Change Certifications can be carried out at the Applicant\'s request.
-2. During a change certification, the Certification Body must determine whether the certified operational environment or component continues to meet the IDS requirements of the originally certified Trust Level after changes have been made.
-3. For small changes, e.g. bug fixes or security patches for components, the change can be certified with minimal effort in form of a maintenance certification.
+1. After a component has been certified, the applicant must inform the Certification Body and the Evaluation Facility (in the case of Assurance Level 2) of any changes made to the component. The applicant must report the changes and fill in the impact assessment provided by the Certification Body.
+2. Change Certifications can also be carried out at the Applicant\'s request.
+3. During a change certification, the Certification Body must determine whether the certified operational environment or component continues to meet the IDS requirements of the originally certified Trust Level after changes have been made.
+4. For small changes, e.g. bug fixes or security patches for components, the change can be certified with minimal effort in form of a maintenance certification.
 For larger changes, e.g. to management processes of an organization or larger source code modifications for a component, a re-certification must be conducted.
 4. For a maintenance certification, the Evaluation Facility evaluates the documented changes to determine whether they may result in a violation of the certified IDS requirements.
 If that is the case, a deeper evaluation in form of a re-certification is required. The above requirements for evaluation apply accordingly except for § 3.3.
 5. For a re-certification, all above requirements for evaluation apply accordingly.
 The scope of the re-certifications shall be such that the changes to the operational environments or components made since the last evaluation are checked.
-Appropriate sampling shall be used to determine whether the operational environment or component as a whole continues to meet the IDS requirements.
+Appropriate sampling shall be used to determine whether the operational environment or component as a whole continues to meet the IDS requirements. The component or operational environment will be assessed using the same certification version (certification criteria and IDS reference testbed version) as for the first evaluation.
```

Comment:
To understand the different scenarios, I believe it is critical to move at least part of number 8 to this statement. Usage of the old versions is directly linked to the validity of the existing certificate not being extended.

Reply:
As we discussed yesterday: If a Change Certification, independent of Maintenance or Re-Certification and independent of the AL, **always** uses the same criteria version as the initial certification, than no change is needed.